app/src/main/java/cl/freedom/wheatermvvm/dashboard/data/CurrentCovid.kt

```diff
@@ -1,9 +1,14 @@
-package cl.freedom.wheatermvvm.data.response
+package cl.freedom.wheatermvvm.dashboard.data
 
 
+import androidx.room.Entity
+import androidx.room.PrimaryKey
 import com.google.gson.annotations.SerializedName
 
-data class Data(
+const val CURRENT_COVID_ID = 0
```

Comment:
All your constants values should go in a file called Constants.kt

Reply:
i've deleted this constant because i changed the logic in database